lib/src/command/add.dart

```diff
@@ -93,7 +93,7 @@ class AddCommand extends PubCommand {
     argParser.addOption('directory',
         abbr: 'C', help: 'Run this in the directory <dir>.', valueHelp: 'dir');
     argParser.addFlag('packages-file',
```

Comment:
Would it make sense to cal this flag `legacy-package-file` just to be extraordinarily clear?